batchflow/models/torch/base.py

```diff
@@ -1402,11 +1401,16 @@ def load(self, path, *args, eval=False, **kwargs):
         else:
             checkpoint = torch.load(path, **kwargs)
 
+        # Save load config values to update `full_config` later
+        load_config = self.config
+        # Remove unnecessary key from load config
+        _ = load_config.pop('build')
```

Comment:
`load_config` is a reference to `self.config`

Reply:
`self.config` is overwritten in line 1410, when corresponding attr is set, but I want to update `full_config` with `self.config`'s values to update `'device'` and save loading path.

I'll remove `pop` from here since it's actually unnecessary and expand the comment